db_stress_tool/db_stress_test_base.cc

```diff
@@ -2248,11 +2313,21 @@ void StressTest::TestCompactFiles(ThreadState* thread,
 
       size_t output_level =
           std::min(random_level + 1, cf_meta_data.levels.size() - 1);
-      auto s = db_->CompactFiles(CompactionOptions(), column_family,
-                                 input_files, static_cast<int>(output_level));
+      CompactionOptions compact_options;
+      if (thread->rand.OneIn(2)) {
+        compact_options.output_file_size_limit = FLAGS_target_file_size_base;
+      }
+      std::ostringstream compact_opt_oss;
+      compact_opt_oss << "output_file_size_limit: "
+                      << compact_options.output_file_size_limit;
+      auto s = db_->CompactFiles(compact_options, column_family, input_files,
+                                 static_cast<int>(output_level));
       if (!s.ok()) {
-        fprintf(stdout, "Unable to perform CompactFiles(): %s\n",
-                s.ToString().c_str());
+        fprintf(stdout,
```

Comment:
This doesn't fail the stress test, so we may need to check crash test stdout to see if the new option cause any non-ok status.

Reply:
Maybe there's a way to permit just the picking conflict failure.

Reply:
Good points - trying to fail stress test for CompactFiles() and CompactRanges() on errors other than injected ones,  file conflict ones and some special ones (e.g, CompactRange`Status::NotSupported("another thread is refitting");`, CompactFiles() input files is deleted due to we call CompactFiles with no lock)...